com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/AOV/AOVRequest.cs

```diff
@@ -172,6 +176,12 @@ public void FillDebugData(DebugDisplaySettings debug)
                 case DebugFullScreen.MotionVectors:
                     debug.SetFullScreenDebugMode(FullScreenDebugMode.MotionVectors);
                     break;
+                case DebugFullScreen.WorldSpacePosition:
+                    debug.SetFullScreenDebugMode(FullScreenDebugMode.WorldSpacePosition);
+                    break;
+                case DebugFullScreen.Fresnel0:
+                    debug.SetDebugViewGBuffer(MaterialDebugSettings.GetGBufferMaterialEnumByName("fresnel0"));
```

Comment:
it is not the correct way of handling this, think about multiple material (hair, fabric etc...)
you should use the debug matching this statement
[MaterialSharedPropertyMapping(MaterialSharedProperty.Specular)]

Reply:
Thanks for the comment, I will fix this as we have discussed.